tutorial/serverless-basic-application/api-gateway.tf

```
@@ -6,22 +6,22 @@ resource "aws_api_gateway_rest_api" "lambda" {
 }
 
 # Resource
-resource "aws_api_gateway_resource" "resource" {
+resource "aws_api_gateway_resource" "app_resource" {
   path_part   = "resource"
   parent_id   = aws_api_gateway_rest_api.lambda.root_resource_id
   rest_api_id = aws_api_gateway_rest_api.lambda.id
 }
 
 # Method
-resource "aws_api_gateway_method" "method" {
+resource "aws_api_gateway_method" "app_method" {
   rest_api_id   = aws_api_gateway_rest_api.lambda.id
   resource_id   = aws_api_gateway_resource.resource.id

```

We would need to update the references of new updated as well, for example:
```suggestion
  resource_id   = aws_api_gateway_resource.app_resource.id
```
Please check and do the same for other references to make sure there is no breakage